.gitignore

```diff
@@ -56,7 +56,9 @@ init.sql
 latest.dump
 
 # snaplet config
-.snaplet
+.snaplet/config.json
```

Comment:
The `.snaplet/config.json` file could stay in the repo.

Reply:
The reason why it would stay is that this is what Snaplet CLI uses to determine where your Snapshots are stored, and other people in your team will be able to run `snaplet snapshot restore` and it'll find the correct snapshots

Reply:
Maybe my config is the old format then, because mine has my personal localhost connection string including my password in it. 🤔 I'll look at the docs and see if that should get changed.

Reply:
Ah, I see! Do you use `.env` files to specify different connection strings on different machines during development? You can also use the `SNAPLET_DATABASE_URL` envar if you don't want to store the connection string:

`SNAPLET_DATABASE_URL=pg://localhost:5432/postgres snaplet proxy`